podpac/core/utils.py

```diff
@@ -475,3 +475,70 @@ def resolve_bbox_order(bbox, crs, size):
     size = size[::r]
 
     return {"lat": [lat_start, lat_stop, size[0]], "lon": [lon_start, lon_stop, size[1]]}
+
+
+def probe_node(node, lat=None, lon=None, time=None, alt=None, nested=False):
```

Comment:
Can you add documentation here? I don't know at a first glance what `active` means, for example, or why `test_composited_prober_nested` has `"inputs": { "inputs": [ ...] }`.

Reply:
This feels to me like it could be a method `Node.probe`.

Reply:
Added documentation and the Node.probe method -- good suggestion.